interface/modules/custom_modules/oe-module-weno/templates/weno_setup.php

```diff
@@ -0,0 +1,164 @@
+                    <div class="row form-group">
+                        <div class="col-12">
+                            <button type="button" class="btn btn-primary btn-refresh float-right" id="app_refresh" onclick="top.location.reload()"
+                                title="<?php echo xla("Same as a browser refresh. Click to implement any new menus and Configuration items."); ?>"><?php echo xlt("Restart OpenEMR"); ?>
+                            </button>
+                            <button type="submit" id="form_save" name="form_save" class="btn btn-primary btn-save float-right d-none" value="Save"><?php echo xlt("Save Setup"); ?></button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </form>
+        <div class="">
+            <div id="accordion" class="accordion">
+                <div class="card mb-1">
+                    <div class="card-header p-1 mb-1 bg-dark text-light collapsed collapsed" data-toggle="collapse" href="#collapseOne">
+                        <h5 class="mb-0"><i class="fa fa-plus mr-2"></i><?php echo xlt("Chores After Setup"); ?></h5>
+                    </div>
+                    <div id="collapseOne" class="card-body collapse" data-parent="#accordion">
+                        <!-- Currently no plans to translate. -->
+                        <?php echo '<h5>' . text('Additional values from Weno for finishing setup are:') . '</h5>' .
+                        text('1. Weno Provider Id: Uxxxx') . '<br />' .
+                        text('2. Assigned Location Id for all the facilities used by the above User Id: Lxxxxx') . '<br />' .
+                        text('3. The provider credentials assign to each prescriber: username(email address) and password.') . '<br /><br />' .
+                        '<h5>' . text('To continue setup follow the below steps.') . '</h5>' .
+                        text('1. Find top menu Admin->Users and select the user associated with the Weno Provider ID Uxxx and enter and save the assigned ID in the Weno Provider ID field.') . '<br />' .
+                        text('2. Find top menu  Admin->Other->Weno Management and enter the assigned Location Id Lxxxxx for the location facilities.') . '<br />' .
+                        text('3. Find top Patient Bar User icon and click Settings. Scroll down or find the Weno button in left sidebar and click. Enter your email and password in the Weno Provider Email and Weno Provider Password fields and Save.') . '<br /><br />' .
+                        '<h5>' . text('Patient Chart Requirements.') . '</h5>' .
+                        text('1. Each Patient is required to have an assigned primary pharmacy from Demographics->Choices. It is good practice to assign an Alternate Pharmacy too.') . '<br />' .
+                        text('2. Each Patient is required to have a Vitals Height and Weight assigned. Create or enter in from an encounter vitals form') . '<br /><br />';
```

Comment:
did you mean for above text calls to be xlt instead?

Reply:
Not sure I want to xlate. thinking..... ... .. .